cli/tests/test_duplicate_command.rs

```diff
@@ -75,7 +75,12 @@ fn test_duplicate() {
 
     let (stdout, stderr) = test_env.jj_cmd_ok(&repo_path, &["undo"]);
     insta::assert_snapshot!(stdout, @"");
-    insta::assert_snapshot!(stderr, @"");
+    insta::assert_snapshot!(stderr, @r#"
+    Undid operation c4b0b2a977fee749cf7d857e915ba5ae70e48e10d2a7d53d76d27efab3184c342eaee24e2bc007c2e6714cdb53f703ae6d20912bd5f541e1b3fcb535bfaa62e6
+    c4b0b2a977fe test-username@host.example.com 2001-02-03 04:05:17.000 +07:00 - 2001-02-03 04:05:17.000 +07:00
+    duplicate 1 commit(s)
+    args: jj duplicate a
```

Comment:
I think we may want a shorter format. I was going to suggest a single-line format similar to `commit_summary`. My concern is just that maybe the `args:` tag is useful enough that we can't really do that? What do you think? I'm thinking something like this (you'll have to imagine colors to separate the different fields):

> Undid operation c4b0b2a9 10 seconds ago, lasted 13 milliseconds duplicate 1 commit(s)

Reply:
I was wondering about that as well. The first line here is quite a mouthful at least (and I just realized this is now printing both the full op id and shortened, so thats redundant either way)

Reply:
If we introduce single-line "operation_summary" (or "op_summary"), we can also update `cmd_op_diff()` to use that. If not, reusing "op_log" template is okay for now.